### PR TITLE
Performance.now(), collect times

### PR DESCRIPTION
**Pull request by rwilliams-scottlogic**
Addresses #2 and #4

### CODE REVIEW

**Comment by chrisprice** [comment]
Any chance you could rebase this onto master?

File: ng-stats.js [hunk 1]
```diff
@@ -114,6 +114,10 @@ var NgStats = function () {
 
 	};
 
+	var now = function () {
+		return performance.now ? performance.now().toFixed(0) : Date.now();
```
Comment (chrisprice) [inline ng-stats.js]: Should this be using round?

File: ng-stats.js [hunk 2]
```diff
@@ -147,6 +151,10 @@ var NgStats = function () {
 				fpsText.textContent = fps + ' W (' + fpsMin + '-' + fpsMax + ')';
 				updateGraph( fpsGraph, bound( 1, 30 - ( fps / ( watchersMax || 2500 ) ) * 30, 30 ) );
 
+				if (digestTimesCollector) {
+					digestTimesCollector.push(ms);
```
Comment (chrisprice) [inline ng-stats.js]: Could it be worth making digestTimesCollector a function?

File: ng-stats.js (hunk 1 above)
Comment (rwilliams-scottlogic) [inline ng-stats.js]: yes

File: ng-stats.js (hunk 2 above)
Comment (rwilliams-scottlogic) [inline ng-stats.js]: I did consider it, would add flexibility. Will make it a digestTimesListener func.

**Comment by rwilliams-scottlogic** [comment]
Are you happy with this now?

**Comment by chrisprice** [comment]
Yes LGTM. Sorry you don't get updates when there's new commits only when there are new comments.

**Pull request merged**